app/javascript/components/Pages/ProductivityCheckLow.js

```diff
@@ -1,26 +1,88 @@
-import React, {Fragment} from 'react';
-import {Button} from "react-bootstrap";
-import {backHandling} from "../helpers/helpers";
-import {BtnBack, BtnNext} from "../UI/ShareContent";
+import React, {useState, Fragment} from 'react';
+import {Footer, Header, Wrapper} from "../UI/ShareContent";
+import flame2 from '../../../assets/images/flame2.svg';
+import flame3 from '../../../assets/images/flame3.svg';
+import flame4 from '../../../assets/images/flame4.svg';
+import flame5 from '../../../assets/images/flame5.svg';
+import flame6 from '../../../assets/images/flame6.svg';
+import flame7 from '../../../assets/images/flame7.svg';
+import flame8 from '../../../assets/images/flame8.svg';
+import flame9 from '../../../assets/images/flame9.svg';
+import flame10 from '../../../assets/images/flame10.svg';
+import { FLAME_IMAGE_SIZES } from '../helpers/consts';
+import {isBlank} from "../helpers/helpers";
 
+const ProductivitySlider = ({productivity, handleSliderChange, flameImages, generateStyles, imageSizes}) => 
+<Fragment>
+    <div className="productivity-container">
+      <form>        
+        <h1 className="mb-2 mx-auto my-0 question">How productive has this week felt to you?</h1>
+        <br />
+        <div className='productivity-card'>     
+          {productivity > 0 && (
+            <div className='image-size'>
+              <img
+                src={flameImages[productivity-1]}
+                alt={`Productivity flame level ${productivity}`}
+                style={generateStyles(imageSizes[productivity - 1])}
+                className="image-container"
+              />
+            </div>          
+          )}          
+          <div className='range-size'>
+            <input
+              type="range"
+              min="0"
+              max="9"
+              defaultValue={productivity}
+              onChange={handleSliderChange}
+              id="productivity-slider"
+              className={`form-range level-${productivity}`}
+            />
+            <p>Use the slider</p>
+          </div>      
+        </div>
+      </form>
+    </div>
+  </Fragment>
 const ProductivityCheckLow = ({data, setData, saveDataToDb, steps, service}) => {
   const {isLoading, error} = service
+  const [productivity, setProductivity] = useState(data.response.attributes.productivity || 0); 
+  
+  
   const handlingOnClickNext = () => {
+    steps.push('ProductivityBadFollowUp');
```

Comment:
rename **ProductivityBadFollowUp** to **productivity-bad-follow-up** to follow selected naming style, and do a renaming where is needed